src/pages/VideoDetail/VideoDetail.css

```diff
@@ -50,3 +50,6 @@
 .video-desc {
```

Comment:
you can add icon-color class here only instead of writing it separately, as both the classes are changing the same property.